rllib/setup.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import
```

Comment:
Maybe add a note at the top explaining that this setup.py is only intended to be used by RLlib developers. Also, maybe update the RLlib documentation for developers to explain how to use this.